### PR TITLE
Update Readme, and allow Mobile param

### PR DESCRIPTION
**Pull request by nickmalcolm**
The readme has been updated for readability and accuracy. The `mobile` param has been supported in the API for a while now, so I've also (hopefully) added that to the SDK properly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,18 +22,9 @@ $apiKey = '<API_KEY>';
...
-
-Alternatively, you can provide more customisation by using an instance of the builder.
-
-```php
-use ThisData\Api\Builder;
-$builder = new Builder($apiKey);
-// Configure the builder here. See the Advanced section below for more details.
-$thisData = $builder->build();
-```
+> :warning: Don't commit your API key is to source control! Use an environment
```
Comment (emarref) [inline README.md]: > API key **in** to source

[sp]

File: src/Endpoint/EventsEndpoint.php [hunk 2]
```diff
@@ -60,9 +61,10 @@ public function trackEvent($verb, $ip, array $user, $userAgent = null)
             self::PARAM_VERB => $verb,
             self::PARAM_IP   => $ip,
             self::PARAM_USER => [
-                self::PARAM_USER__ID    => $this->findValue(self::PARAM_USER__ID, $user),
-                self::PARAM_USER__NAME  => $this->findValue(self::PARAM_USER__NAME, $user),
-                self::PARAM_USER__EMAIL => $this->findValue(self::PARAM_USER__EMAIL, $user),
+                self::PARAM_USER__ID     => $this->findValue(self::PARAM_USER__ID, $user),
+                self::PARAM_USER__NAME   => $this->findValue(self::PARAM_USER__NAME, $user),
+                self::PARAM_USER__EMAIL  => $this->findValue(self::PARAM_USER__EMAIL, $user),
+                self::PARAM_USER__MOBILE => $this->findValue(self::PARAM_USER__MOBILE, $user),
```
Comment (emarref) [inline src/Endpoint/EventsEndpoint.php]: Consider updating `EventsEndpointTest` to confirm mobile comes through.

**Comment by emarref** [comment]
A couple of non-blocking comments. Otherwise :shipit:.

**Comment by nickmalcolm** [comment]
I made the tests pass 😎 

```
thisdata-php$ ./vendor/bin/phpunit
PHPUnit 5.3.4 by Sebastian Bergmann and contributors.

Runtime:       PHP 5.6.21
Configuration: /Users/nickmalcolm/workspace/thisdata-php/phpunit.xml

................................                                  32 / 32 (100%)

Time: 84 ms, Memory: 6.25MB
```


Feels a bit weird the way those expectations are set up. For example, it's hard to test that sending just a user ID works, because the helper method is checking that user email and name are also present. That's my 2c.

**Pull request merged**